src/renderer/scss/_gui.scss

```diff
@@ -252,10 +252,11 @@ p {
 }
 
 .credit-amount {
-  padding: 5px;
```

Comment:
Just keep this and change the font-size. It seems to look fine.

Reply:
PR is updated now.